src/PageLayout/useStickyPaneHeight.ts

```diff
@@ -44,11 +46,11 @@ export function useStickyPaneHeight() {
       // We need to account for this when calculating the offset.
       const overflowScroll = Math.max(window.scrollY + window.innerHeight - document.body.scrollHeight, 0)
 
-      calculatedHeight = `calc(100vh - ${topOffset + bottomOffset - overflowScroll}px)`
+      calculatedHeight = `calc(100vh - ${Math.max(topOffset, stickyTopHeight) + bottomOffset - overflowScroll}px)`
```

Comment:
🆘  Re @vdepizzol 's [comment](https://github.com/primer/react/pull/2212#discussion_r941751072) about rem on the parent PR, I see that this logic works with the pane height being px? and how do we plan to move it forward to make it with rem? Sorry, I couldn't gather much about our approach for css units by looking at the codebase 😞  cc @colebemis

Reply:
I got string values for `stickyTop` working in this commit: [2d429d1](https://github.com/primer/react/pull/2232/commits/2d429d162b467a52f167535fba131fbf4af6b54e) 🎉  Let me know if you notice any issues with that implementation.